src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+:root {
+  --light-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-right: 50px;
+}
+
+.header__icon a {
+  margin: auto 50px;
+  display: block;
+}
+
+.header__links {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.header__links li {
+  position: relative;
+}
+
+.header__links li:nth-child(n+2) {
+  margin-left: 20px;
+}
+
+a.is-active {
+  color: var(--light-blue) !important;
```

Comment:
do not use `!important`. it's better to increase the selector weight - [here is the information](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)